app/views/assessments/viewFeedback.html.erb

```diff
@@ -1,120 +1,40 @@
-</h2>
-<div class="result-container">
-  <div class="feedback-container">
-    <div class="result-pane">
-      <div class="feedback-header">
-        <h3>Feedback</h3>
-        <div class="feedback-status__completed">
-          Completed
-        </div>
-      </div>
-      <pre><%= @score.feedback %></pre>
-    </div>
-    <% if @submission.annotations.count > 0 and (@cud.instructor? or @cud.course_assistant? or @problemReleased) %>
-    <div class="result-pane">
-      <div class="feedback-header">
-        <h2 style="margin-top: 1em">Remarks</h2>
-      </div>
-      <% subs_by_problem = @submission.annotations.group_by(&:problem)
-          subs_by_problem.each do |problem, annotations| %>
-          <b> <%= problem.nil? ? "Deleted Problem(s)" : problem.name %> </b>
-          <ul class="collection">
-          <%# use custom sort b/c using ordering doesn't work with autograder output, archived files %>
-            <%
-              s_annotations = annotations.sort do |a,b|
-              aFilename = @get_correct_filename.call(a)
-              bFilename = @get_correct_filename.call(b)
-
-              if a.global_comment? && b.global_comment?
-                a.id <=> b.id
-              else
-                case
-                  when a.global_comment? != b.global_comment?
-                    a.global_comment? ? -1 : 1
-                  when aFilename != bFilename
-                    aFilename <=> bFilename
-                  else
-                    a.line <=> b.line
-                  end
-                end
-              end
-              s_annotations.each do |annotation| %>
-                <li class="collection-item">
-                  <div style='white-space: pre-wrap;'><%= annotation.comment%> </div>
-                  <% filename = @get_correct_filename.call(annotation)%>
-                  <span> 
-                    <b><%= annotation.value? ? sprintf("(%+.2f)", annotation.value.round(2)) : "" %></b>
-                    <% if annotation.global_comment? %>
-                      <%= link_to "Global Annotation", [:view, @course, @assessment, @submission] %>
-                    <% else %>
-                      <%# line + 1 because line numbers start from 1 %>
-                      <%= link_to filename.to_s + ":" + (annotation.line + 1).to_s, [:view, @course, @assessment, @submission, header_position: annotation.position] %>
-                    <% end %>
-                  </span>
-                </li>
-              <% end %>
-            </ul>
+  <h2>Autograding Result for 
+      <%= @score.nil? ? @assessment.display_name : "#{@assessment.display_name} - #{@score.problem.name} (#{@score.submission.course_user_datum.user.email})"%> 
+  </h2>
+  <div class="result-container">
+    <div class="feedback-container">
+      <div class="result-pane">
+        <% if @score.nil? %>
+          <div class="feedback-header">
+            <h3>Feedback</h3>
+            <div id="status-refresh">
```

Comment:
Like I mentioned in the other comment, the refresh button should "spin" when you click on it.
This is important because sometimes when a user refresh, but nothing is getting outputted from the course autograder. So they don't know if the click of the refresh is being registered.

For an example of the refresh button behavior, refer to the refresh button on the metrics page http://localhost:3001/courses/AutoPopulated/metrics/index

Reply:
done

Reply:
For some reason, the refresh icon at the top rotates off-center (at least, sometimes). The one at the bottom is fine though.

https://user-images.githubusercontent.com/9074856/205424604-c0352319-42a2-445f-8a20-6abf706c3aed.mov